src/App.js

```diff
@@ -1,12 +1,101 @@
 import React, { Component } from 'react';
 import './App.css';
+import Article from './Article';
 import Header from './Header';
+import Footer from './Footer';
 
 class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      selected: 0,
+      article_expanded: false,
```

Comment:
Use camelcase instead of underscores in Javascript code (different style pattern than Python):

e.g. `articleExpanded: false`